packages/core/core.ts

```diff
@@ -433,15 +517,18 @@ export function wrapFuzzFunctionForBugDetection(
 			done: (err?: Error) => void
 		): void | Promise<void> => {
 			try {
+				hooking.hookManager.runBeforeEachCallbacks();
 				// Return result of fuzz target to enable sanity checks in C++ part.
 				return originalFuzzFn(data, (err?: Error) => {
```

Comment:
Can this `return` be pulled out of the `try`? It looks like it can't fail, only `runBeforeEachCallbacks` can.

Reply:
I pulled it out!